Just/Just.swift

```diff
@@ -901,7 +901,20 @@ public final class HTTP: NSObject, NSURLSessionDelegate, JustAdaptor {
                         task.resume()
                     }
                     if isSync {
-                        dispatch_semaphore_wait(semaphore, DISPATCH_TIME_FOREVER)
+                        // Express one second in nanoseconds.
+                        let NSEC: Int64 = 1000000000
```

Comment:
Thanks - I have actually figured out that this won't work (as intended) because converting *before* multiplying causes loss of precision.
`NSEC_PER_SEC` is a `UInt64` too, so would have the same problem.
Changed my code to:
```
let timeoutNanosecs: Int64 = Int64(tint * 1000000000.0)
```
Not great, admittedly, but given that Swift isn't smart enough to allow a multiplication between an `Int64` and a `Double`, I'm afraid we're stuck with this?
(I have only coded in Swift for less than 3 weeks - so very happy to get suggestions here!)